Wrong partition order being created

The example of [boot-raid1](https://github.com/nix-community/disko/blob/master/example/boot-raid1.nix) should be created with the type `EF02` partition being first then the type `EF00` partition.

But instead, it is *reversed* with the following output (with some removed unneeded whitespace and modified for single disk) being created:

```sh
#! /nix/store/vqvj60h076bhqj6977caz0pfxs6543nb-bash-5.2-p15/bin/bash
export PATH=/nix/store/aj8lqifsyynq8iknivvxkrsqnblj7qzs-jq-1.6-bin/bin:/nix/store/lmqgs26nq3q1qzjnhvm6c00x9jlklkcm-gptfdisk-1.0.9/bin:/nix/store/fk94q6xm7h3bzvlla5zk83b0f3sm25jk-systemd-minimal-253.6/bin:/nix/store/gz5rdraspl7mx42j07f9zjsa3cqzc26r-parted-3.6/bin:/nix/store/3acmz0y6v5y22kkivp798wz6yaggbyxw-util-linux-2.39.2-bin/bin:/nix/store/773qwyfis4fic1flgc8vl18cn8lmdvlk-dosfstools-4.2/bin:/nix/store/vqvj60h076bhqj6977caz0pfxs6543nb-bash-5.2-p15/bin:$PATH
umount -Rv "/mnt" || :

# shellcheck disable=SC2043
for dev in /dev/sda; do
  /nix/store/qds532rmhjnmhdcj63zxvfnqc8qrrzqz-disk-deactivate/disk-deactivate "$dev"
done

set -efux

disko_devices_dir=$(mktemp -d)
trap 'rm -rf "$disko_devices_dir"' EXIT
mkdir -p "$disko_devices_dir"

( # disk one /dev/sda   #
  device='/dev/sda'
  imageSize='2G'
  name='one'
  type='disk'
  
  ( # gpt  /dev/sda   #
    device='/dev/sda'
    type='gpt'
    
    sgdisk \
      --set-alignment=2048 \
      --align-end \
      --new=1:0:+100M \
      --change-name=1:disk-one-ESP \
      --typecode=1:EF00 \
      /dev/sda
    # ensure /dev/disk/by-path/..-partN exists before continuing
    partprobe /dev/sda
    udevadm trigger --subsystem-match=block
    udevadm settle
    ( # filesystem  /dev/disk/by-partlabel/disk-one-ESP vfat /boot #
      device='/dev/disk/by-partlabel/disk-one-ESP'
      declare -a extraArgs=()
      format='vfat'
      declare -a mountOptions=('defaults')
      mountpoint='/boot'
      type='filesystem'
      
      mkfs.vfat \
         \
        /dev/disk/by-partlabel/disk-one-ESP
    )
    
    sgdisk \
      --set-alignment=2048 \
      --align-end \
      --new=2:0:+1M \
      --change-name=2:disk-one-boot \
      --typecode=2:EF02 \
      /dev/sda
    # ensure /dev/disk/by-path/..-partN exists before continuing
    partprobe /dev/sda
    udevadm trigger --subsystem-match=block
    udevadm settle
  )
)


set -efux
# first create the necessary devices


# and then mount the filesystems in alphabetical order
if ! findmnt /dev/disk/by-partlabel/disk-one-ESP "/mnt/boot" >/dev/null 2>&1; then
  mount /dev/disk/by-partlabel/disk-one-ESP "/mnt/boot" \
    -t "vfat" \
    -o defaults \
    -o X-mount.mkdir
fi
```

--- a/example/boot-raid1.nix
+++ b/example/boot-raid1.nix
@@ -7,7 +7,7 @@
         content = {
           type = "gpt";
           partitions = {
-            boot = {
+            Boot = {
               size = "1M";
               type = "EF02"; # for grub MBR
             };

--- a/example/boot-raid1.nix
+++ b/example/boot-raid1.nix
@@ -7,7 +7,7 @@
         content = {
           type = "gpt";
           partitions = {
-            Boot = {
+            BOOT = {
               size = "1M";
               type = "EF02"; # for grub MBR
             };

